firmware/common/config/system/group1_system.inc

```diff
@@ -31,8 +31,10 @@ GROUP 1 System
     FUNCTION 2 Key Status
         i1 = *DPARAMS;
         *DPARAMS = (i1 < KBD_MAX_KEYCODE) ? KBDGetStateArray()[i1] : 0;
```

Comment:
I would suggest to change *DPARAMS to DPARAMS[0] to match the style in the next line.

Reply:
fixed